boilerplate/_lib/container-make

```diff
@@ -12,15 +12,18 @@ source ${0%/*}/common.sh
 CONTAINER_ENGINE=$(command -v podman || command -v docker)
 [[ -n "$CONTAINER_ENGINE" ]] || err "Couldn't find a container engine. Are you already in a container?"
 
+CONTAINER_ROOT="/go/src/${REPO_ROOT##*src}"
```

Comment:
- Can we rely on `REPO_ROOT` having a `src` component? I don't know how boldly we get to assert "your go directory setup is bogus" if not.
- On that note, should we have a fallback (to the old behavior maybe) if that pattern fails?
- To be safe (paranoid), perhaps we should include the slashes
```suggestion
CONTAINER_ROOT="/go/src/${REPO_ROOT##*/src/}"
```
...you know, in case some repository happens to have the substring `src` in its name.

Reply:
> * Can we rely on `REPO_ROOT` having a `src` component? I don't know how boldly we get to assert "your go directory setup is bogus" if not.

Update on this: apparently it is not only *not* bogus, it is *recommended* not to put your sources under `GOPATH/src` when using modules (for reasons I don't fully understand). So yeah, we need to come up with something different here.